_lessons/packages.html

```diff
@@ -61,9 +61,22 @@
 ```shell
 $ dbt deps
 ```
+---
+## demo
+
+<br>
+<br>
+.center[
+<img src="/ui/img/dbt-standalone-new.png" style="width: 20%;">
+]
+
 ???
+Demo Time:
+* add packageby navigating to https://hub.getdbt.com/fishtown-analytics/dbt_utils/latest/ and copying yml
```

Comment:
typo: add a space to "packageby". Also let's update the URL to dbt-labs instea of fishtown